client/app/kubernetes/nodes/kubernetes-nodes.html

```diff
@@ -31,7 +31,7 @@
                       data-translate="{{:: 'kube_service_cluster_status_' + $row.status }}">
                 </span>
             </oui-column>
-            <oui-action-menu data-disabled="$row.status === 'INSTALLING' || $row.status === 'DELETING'"  data-align="end" data-compact>
+            <oui-action-menu data-disabled="$row.status === 'INSTALLING' || $row.status === 'DELETING' || $ctrl.cluster.status === 'RESETTING'" data-align="end" data-compact>
```

Comment:
Duplicated code with https://github.com/ovh-ux/ovh-manager-cloud/pull/1268/files#diff-2b1843a9c04498d4d8156544bb90e834R31

This should be refactored in a function :)